lib/services/tokens/token.service.js

```diff
@@ -144,7 +145,24 @@ s.revoke = function (accessToken) {
   });
 };
 
-function createInternalToken (criteria, id, token, type) {
+s.createJWT = function (payload) {
+  const accessTokenConfig = config.systemConfig.accessTokens;
+  const secret = accessTokenConfig.secretOrPriKeyFile ? fs.readFileSync(accessTokenConfig.secretOrPriKeyFile) : accessTokenConfig.secretOrPriKey;
```

Comment:
1) Pri = Private ? 
2) why not resolve it in policy\plugin and make service accept it as parameter? 
current implementation will reread the file on each JWT generation

Reply:
1. Yes, it's private. It's probably a very bad combination, I'm open to better ideas.
2. Definitely good point — I recall I was having some problems with that this morning (c2d30d1) but I forgot to go back on it. 👍

Reply:
Private seems to be good word ? why not call it that ?

Reply:
Ah good question — it's because in the JWT policy the equivalent parameter is called `secretOrPubKeyFile`, and I wanted to kind to stick to the convention.

Reply:
since it is not released, what if we change `pub` to `public`  ? 
I believe it will be easier to maintain code

Reply:
If @kevinswiber agrees (as he was the one suggesting the name) I am ok with the name change.

Reply:
I would prefer to be explicit and obvious with longer parameter names.  `Pub` => `Public`, `Pri` => `Private`

Reply:
Done in 93b0f1d

Reply:
@DrMegavolt N.2 resolved in 58e3ce81aea8206134ce1c8b4ff79b64dd401f23